docs/pages/router/overview.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Expo Router
+description: Welcome to Expo Router
+sidebar_title: Overview
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+Expo Router is a routing library for React Native applications built with Expo. It provides a way to manage navigation between screens in your app, allowing users to move between different parts of your app's UI seamlessly, using the same components on multiple platforms (iOS, Android, Web, etc).
+
+Expo Router brings the best routing concepts from the web to native iOS and Android apps. Every file in the **app** directory automatically becomes a route in your mobile navigation, making it easier than ever to build, maintain, and scale your project. It's built on top of our powerful [React Navigation suite](https://reactnavigation.org/) enabling truly native navigation. The entire deep linking system is automatically generated live, so users can share links to any route in your app.
```

Comment:
```suggestion
Expo Router brings the best file-system routing concepts from the web to native Android and iOS apps. When a file is added to the **app** directory, it automatically becomes a route in your mobile navigation. It makes building, maintaining, and scaling your project easier than ever. It's built on top of our powerful [React Navigation suite](https://reactnavigation.org/), enabling truly native navigation. The entire deep linking system is automatically generated live so that users can share links to any route in your app.
```

Just introducing the concept of "every file in **app** directory" doesn't make sense to me. I've left a suggestion with some re-phrasing and a little bit more context.